NSObject+Rx.podspec

```diff
@@ -18,5 +18,5 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/RxSwiftCommunity/NSObject-Rx.git", :tag => s.version }
   s.source_files  = "*.swift"
   s.frameworks  = "Foundation"
-  s.dependency 'RxSwift', '~> 3.0'
+  s.dependency 'RxSwift', '~> 4.0.0'
```

Comment:
Can we keep this as `~> 4.0` instead of `4.0.0` to accommodate version 4.1, 4.2 etc?

Reply:
Yes 👍